modules/nux/modules.php

```diff
@@ -592,12 +597,12 @@ protected function output() {
                 $res .= sprintf(' <a href="?page=servers#%s_section">%s</a>', $proto, $this->trans('Add'));
             }
             else {
-                if ($server_data[$proto] > 1) {
```

Comment:
You will need this if check to support the s for exemple source and sources

Reply:
```
It is already resolved here ($server_data[$proto] > 1 ? 'sources' : 'source') and here ($server_data[$proto] > 1 ? 'sources' : 'source').
```

Reply:
It should be: You have 1 IMAP source(sources if many) | You have 1 JMAP source(sources if many)
<img width="581" alt="Screenshot 2024-11-04 at 07 18 35" src="https://github.com/user-attachments/assets/191d3e85-2393-4d53-a655-e61f5ffa3305">

Reply:
<img width="778" alt="image" src="https://github.com/user-attachments/assets/eb8ac9da-341c-48ea-b6a2-45458b2f7b76">